tensorflow/lite/c/CMakeLists.txt

```diff
@@ -36,6 +36,30 @@ add_subdirectory(
 
 set(CMAKE_CXX_STANDARD 11)
 
+if (CMAKE_SIZEOF_VOID_P EQUAL 4)
+  set(64_BIT FALSE)
+elseif (CMAKE_SIZEOF_VOID_P EQUAL 8)
+  set(64_BIT TRUE)
+endif()
+
+if (MSVC AND (NOT 64_BIT))
+  message("Disabling MSVC /O2 optimization")
+  set(CompFlags
+    CMAKE_CXX_FLAGS_RELEASE
```

Comment:
Will this work with Debug build?

Reply:
No need to disable /O2 and enable /O1 for debug build, since optimisation is disabled by default (/Od) when building Debug libraries